src/evotorch/_distribute.py

```
@@ -0,0 +1,1221 @@
+            A tuple in the form `(task_index, result)` where `task_index` is
+            the index of the task that was given, and `result` is the result
+            of the wrapped function, moved back to the cpu.
+        """
+
+        if self.is_main:
+            raise RuntimeError("This function should not be executed from the main actor")
+
+        num_explicit_devices = len(self.__devices)
+        prepared_args = []
+
+        for split_arg, arg in zip(self._iter_split_arguments(args), args):
```

Ruff (0.14.0)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)